server/main.go

```diff
@@ -8,12 +8,14 @@ import (
 	"net/http"
 	"school-project-2019/server/domain"
 	"school-project-2019/server/domain/devices"
-
 	"github.com/rs/cors"
+	"github.com/subosito/gotenv"
 	//"school-project-2019/server/storage"
 )
 
 func main() {
+	gotenv.Load()
```

Comment:
Did you added the env file?
If it should be created for all developers - please add in README, how to do it.

Reply:
I'd suggest adding `.env.example` file with dummy values but proper keys